src/styles/block/menu.scss

```diff
@@ -0,0 +1,67 @@
+.menu {
```

Comment:
When I use 'tab' - I recieved a vertical scroll, please, fix it. Position and z-index can help you with this issue.